docs/source/migrations/modes.rst

```diff
@@ -0,0 +1,46 @@
+======================
+Snuba Migration Modes
+======================
+
+The main "switch" between the two modes for running data migrations (local and
+distributed) lives in a user's local Sentry configuration (in their ``sentry.conf.py``).
+The controlling envrionment variable is ``SENTRY_DISTRIBUTED_CLICKHOUSE_TABLES``,
+and its default value is set `at this line <https://github.com/getsentry/sentry/blob/master/src/sentry/conf/server.py#L127>`_.
+
+Once this boolean variable is set, one of two Clickhouse Docker containers will be
+(allowed to) start, depending on the mode (distributed or local). Whenever a user
+wants to switch between the two modes, they must "turn off" the existing running
+container, alter the environment variable mentioned above, and "turn on" the
+correct container to be in the new mode.
+
+More information on migrations in general can be found `here <https://github.com/getsentry/snuba/blob/master/MIGRATIONS.md>`_.
+
+Enabling Local Mode
+=====================
+
+In your local Sentry environment configuration, set ``SENTRY_DISTRIBUTED_CLICKHOUSE_TABLES``
+to False. Start up the corresponding Clickhouse container (``sentry devservices up clickhouse``).
+
+Ensuring that your local configuration is prepared for migrations (reverting any existing
+migrations, or dropping certain databases/tables), run migrations as expected
+(``snuba migrations migrate --force``).
+
+
+Enabling Distributed Mode
```

Comment:
Could you add a note that this is experimental and isn't yet supposed to work properly? Since there are quite a number of known bugs, we should prevent anyone from attempting to use this for anything other than dev yet.

Reply:
Will do, once the new switch is confirmed.